cmd/telemetrygen/pkg/metrics/config.go

```diff
@@ -34,11 +36,13 @@ func (c *Config) Flags(fs *pflag.FlagSet) {
 
 	fs.StringVar(&c.HTTPPath, "otlp-http-url-path", c.HTTPPath, "Which URL path to write to")
 
-	fs.Var(&c.MetricType, "metric-type", "Metric type enum. must be one of 'Gauge' or 'Sum'")
 	fs.IntVar(&c.NumMetrics, "metrics", c.NumMetrics, "Number of metrics to generate in each worker (ignored if duration is provided)")
 
 	fs.StringVar(&c.TraceID, "trace-id", c.TraceID, "TraceID to use as exemplar")
 	fs.StringVar(&c.SpanID, "span-id", c.SpanID, "SpanID to use as exemplar")
+
+	fs.Var(&c.MetricType, "metric-type", "Metric type enum. must be one of 'Gauge' or 'Sum'")
+	fs.Var(&c.TemporalityType, "temporality-type", "Temporality for Sum and Histogram metrics. Must be one of 'delta' or 'cumulative'")
```

Comment:
Let's call this "aggregation temporality" as this is the terminology used in the [data model](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#sums):
```suggestion
	fs.Var(&c.AggregationTemporality, "aggregation-temporality", "Aggregation temporality for metrics. Must be one of 'delta' or 'cumulative'")
```

I also omitted the metric types, since we may also add exponential histograms to the mix in the future.

Reply:
Thanks for the suggestion. Makes sense. Addressed it [aba04c2](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/38146/commits/aba04c25b00a5e2f9b1e96d9fa5a8b25f61ebd00)